lib/ProductOpener/Ingredients.pm

```diff
@@ -2532,9 +2533,16 @@ sub init_percent_values ($total_min, $total_max, $ingredients_ref) {
 				$ingredient_ref->{percent_min} = 0;
 			}
 			if ((not defined $ingredient_ref->{percent_max}) or ($ingredient_ref->{percent_max} > $total_max)) {
-				$ingredient_ref->{percent_max} = $total_max;
+				my $value = get_inherited_property("ingredients", $ingredient_ref->{id}, "percent_max:en");
+				if (defined $value and $index > 0) {
```

Comment:
I could not construct a test case where it made a difference, but I'd feel safer if we also tested that $value < $total_max